pkg/expr/identity.go

```diff
@@ -45,12 +47,53 @@ func (identity) EvalString(mapExpression string, bag attribute.Bag) (string, err
 	return "", fmt.Errorf("%s not in attribute bag", mapExpression)
```

Comment:
Is this a message that can make it out to operators? If so, it shouldn't use the term "attribute bag"